projects/plugins/jetpack/modules/likes/jetpack-likes-settings.php

```diff
@@ -224,11 +236,10 @@ function sharing_block() { ?>
 			</table>
 
 			<p class="submit">
-				<input type="submit" name="submit" class="button-primary" value="<?php esc_attr_e( 'Save Changes', 'jetpack' ); ?>" />
-			</p>
-
-			<input type="hidden" name="_wpnonce" value="<?php echo wp_create_nonce( 'sharing-options' );?>" />
-		</form> <?php
+			<input type="submit" name="submit" class="button-primary" value="<?php esc_attr_e( 'Save Changes', 'jetpack' ); ?>" />
+			<input type="hidden" name="_wpnonce" value="<?php echo wp_create_nonce( 'sharing-options' ); // phpcs:disable -- WordPress.Security.EscapeOutput.OutputNotEscaped ?>" />
```

Comment:
Why not just escape it? Or even better, use `wp_nonce_field()`.
```suggestion
			<?php wp_nonce_field( 'sharing-options' ); ?>
```